Sources/DB/DB.swift

```diff
@@ -12,15 +12,53 @@
 import Foundation
 import RealmSwift
 
+fileprivate var realmMigrated = false
+
 /// returns a configured realm object.
 func getRealm() throws -> Realm {
     return try Realm(configuration: config)
 }
 
+fileprivate func migrateTrackInfoObjects(_ migration: Migration) {
+    
+    migration.enumerateObjects(ofType: DTGItemRealm.className()) { (oldObj, newObj) in
+        guard let newObj = newObj, let oldObj = oldObj else {return}
+
+        let names = [
+            (TrackInfo.TrackType.text, "selectedTextTracks", "textTracks"), 
+            (TrackInfo.TrackType.audio, "selectedAudioTracks", "audioTracks")
+        ]
+        
+        for (type, oldName, newName) in names {
+
+            let oldTracks = oldObj[oldName] as! List<DynamicObject>
+            let newTracks = List<TrackInfoRealm>()
+            
+            for t in oldTracks {
+                let ti = TrackInfo(languageCode: t["languageCode"] as! String, title: t["title"] as! String, type: type)
+                let tir = TrackInfoRealm(itemId: oldObj["id"] as! String, selected: true, trackInfo: ti)
```

Comment:
Let's try to solve this in a different way, without '!'.
Don't want it to crash by any chance.